app/models/katello/system.rb

```diff
@@ -56,9 +56,9 @@ class System < Katello::Model
     scoped_search :in => :fact_values, :on => :value, :in_key => :fact_names, :on_key => :name, :rename => :facts, :complete_value => true,
                   :only_explicit => true, :ext_method => :search_cast_facts
     scoped_search :on => :description, :complete_value => true
-    scoped_search :in => :host_collections, :on => :name, :complete_value => true, :rename => :host_collection
+    scoped_search :in => :foreman_host, :on => :name, :complete_value => true, :rename => :host
     scoped_search :in => :environment, :on => :name, :complete_value => true, :rename => :environment
-
+    scoped_search :in => :host_collections, :on => :name, :rename => :host_collection, :complete_value => true
```

Comment:
this doesn't actually seem to be working.  its auto completing fine, but searching is using the wrong query:

```
FROM         "katello_systems"
        INNER JOIN "katello_host_collection_hosts"
        ON "katello_systems"."id" = "katello_host_collection_hosts"."host_id" 
        INNER JOIN "katello_host_collections"
        ON "katello_host_collection_hosts"."host_collection_id" = "katello_host_collections"."id" 
 WHERE "katello_host_collections"."name" = 'both' )))

```